transformer/request.py

```diff
@@ -51,6 +51,7 @@ class Request(NamedTuple):
     timestamp: datetime
     method: HttpMethod
     url: SplitResult
+    name: str
```

Comment:
Is there a particular reason for adding this field here rather than at the end of the list (after `query`)?

Otherwise, since none of these fields are optional, I think it would be safer to add `name` at the end, so that no code using the Request constructor without keyword arguments needs to be changed.

Also, since this name is optional, I would rather see `name: Optional[str] = None`, so that we don't have to specify it when we don't need it.

Reply:
thinks for your review, put name to end of list is a good suggestion, i will fix it and add more test to test_locust.py.

Reply:
@thilp I adjusted codes, and add test update task name in plugin. https://github.com/zalando-incubator/Transformer/pull/34/commits/950e7323f556d5a9fe0d897e8815b15e7859f322

Reply:
Thank you!